functions/HttpTriggerAPIUsersId/__init__.py

```diff
@@ -160,25 +161,32 @@ def get_user(conn, user_id, _redis):
 
 
 def get_user_cache(_redis):
-    try:
-        cache = _redis.get(ALL_USERS_KEY)
-        return cache
-    except TypeError as e:
-        logging.critical('Failed to fetch user from cache: ' + e.args[1])
-        return None
+    if CACHE_TOGGLE == "true":
```

Comment:
If you just remove the `== "true"` this will work better - sometimes it is True, sometimes true, in the Azure function app, but both of those will evaluate to True if you don't tell it to compare Strings. If that makes sense. So everywhere you have `if CACHE_TOGGLE == "true":` to just `if CACHE_TOGGLE:`